### ISSUE TITLE
Add Attenuation Model with not attenuation

### ISSUE THREAD

**Issue by NilsIrl** [godotengine/godot#26538]
The Idea behind this is that sometimes only the direction from which the sound comes from is important in which case it can be nice if the attenuation can be rules out of the equation.

This can be useful to make a sound source that move and where the developer chooses how attenuation works.

For the moment the only way to deal with this is to do code that goes against the already existing one which is quiet a waste of time (performance + developer time)

I'll be very happy to do a pull request but I first want to make sure this is a wanted feature or at least one that will be pulled in.

**Comment by NilsIrl** [comment]
For the moment the way I have found to deal with this problem without having a custom engine (which I don't want to and is why I am submitting this issue) is to use the following line of codes.
```gdscript
var distance = $AudioStreamPlayer3D.global_transform.origin.distance_to(player.global_transform.origin)
$AudioStreamPlayer3D.unit_db = INITIAL_DB - linear2db(1 / (distance / $AudioStreamPlayer3D.unit_size))
```
which is exactly the opposite of what https://github.com/godotengine/godot/blob/329904598829dcbed462b012dee39bc37d7d7c83/scene/3d/audio_stream_player_3d.cpp#L217 is doing.

Another reason which I didn't mention before for why I would be interesting to have this is that it would allow to have the sound attenuation be against another object instead of the actual AudioStreamPlayer3D.

**Comment by NilsIrl** [comment]
#26868 Was submitted to fix this problem.

**Comment by akien-mga** [comment]
Fixed by #26868.
